tmk_core/protocol/nrf/sdk15/usbd.c

```diff
@@ -736,6 +744,34 @@ int usbd_enable(void) {
 
 void usbd_process(void) {
   while (app_usbd_event_queue_process()) {
+    if (auto_connection_check) {
+      if (auto_connection_check_cnt > 100) {
+        auto_connection_check = false;
+        NRF_LOG_DEBUG("auto_connection_check!");
+        NRF_LOG_DEBUG("nrf_drv_usbd_is_started: %d", nrf_drv_usbd_is_started());
+        NRF_LOG_DEBUG("nrf_drv_usbd_bus_suspend_check: %d", nrf_drv_usbd_bus_suspend_check());
+        if (nrf_drv_usbd_is_started()) {
+          if (!get_usb_enabled()) {
+            if (!nrf_drv_usbd_bus_suspend_check()) { // connect to USB HOST
+              select_usb(true);
```

Comment:
Why is `select_usb` given an argument here? The header `tmk_core/protocol/nrf/app_ble_func.h` defines it without an argument.